ovh/resource_cloud_project_region_loadbalancer_log_subscription.go

```diff
@@ -0,0 +1,187 @@
+				Required:    true,
+				ForceNew:    true,
+			},
+			"loadbalancer_id": {
+				Type:        schema.TypeString,
+				Description: "ID representing the loadbalancer of the resource",
+				Required:    true,
+				ForceNew:    true,
+			},
+			"stream_id": {
+				Type:        schema.TypeString,
+				Description: "ID representing the stream of the resource",
+				Required:    true,
+				ForceNew:    true,
+			},
+			"kind": {
+				Type:        schema.TypeString,
+				Description: "Log kind name of this subscription",
+				Required:    true,
+				ForceNew:    true,
+			},
+
+			//computed
+			"created_at": {
+				Type:        schema.TypeString,
+				Description: "Creation date of the subscription",
+				Computed:    true,
+			},
+			"ldp_service_name": {
+				Type:        schema.TypeString,
+				Description: "Name of the destination log service",
+				Sensitive:   true,
+				Computed:    true,
+			},
+			"operation_id": {
+				Type:        schema.TypeString,
+				Description: "Identifier of the operation",
+				Computed:    true,
+			},
+			"resource_name": {
+				Type:        schema.TypeString,
+				Description: "Name of subscribed resource, where the logs come from",
+				Computed:    true,
+			},
+			"resource_type": {
+				Type:        schema.TypeString,
+				Description: "Type of subscribed resource, where the logs come from",
+				Computed:    true,
+			},
+			"updated_at": {
+				Type:        schema.TypeString,
+				Description: "Last update date of the subscription",
+				Computed:    true,
+			},
+		},
+	}
+}
+
+func resourceCloudProjectRegionLoadbalancerSubscriptionsCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	config := meta.(*Config)
+	serviceName := d.Get("service_name").(string)
+	regionName := d.Get("region_name").(string)
+	loadbalancerID := d.Get("loadbalancer_id").(string)
+
+	endpoint := fmt.Sprintf("/cloud/project/%s/region/%s/loadbalancing/loadbalancer/%s/log/subscription",
+		url.PathEscape(serviceName),
+		url.PathEscape(regionName),
+		url.PathEscape(loadbalancerID),
+	)
+	params := (&CloudProjectRegionLoadbalancerLogSubscriptionResourceCreateOpts{}).fromResource(d)
+	res := &CloudProjectRegionLoadbalancerLogSubscriptionResponse{}
+
+	log.Printf("[DEBUG] Will create Log subscrition : %+v for loadbalancer %s on region %s from project %s", params, loadbalancerID, regionName, serviceName)
+	err := config.OVHClient.Post(endpoint, params, res)
+	if err != nil {
+		diag.Errorf("calling Post %s with params %+v:\n\t %q", endpoint, params, err)
+	}
+
+	log.Printf("[DEBUG] Waiting for Log subscription operation %s to be READY", res.OperationID)
+	op, err := waitForDbaasLogsOperation(ctx, config.OVHClient, res.LDPServiceName, res.OperationID)
+	if err != nil {
+		return diag.Errorf("timeout while waiting log subscrition operation %s to be READY: %q", res.OperationID, err)
+	}
+	log.Printf("[DEBUG] Log subscrition operation %s is READY", res.OperationID)
```

Comment:
```suggestion
	log.Printf("[DEBUG] Log subscription operation %s is READY", res.OperationID)
```

```
$:~/go/src/github.com/ovh/terraform-provider-ovh GIT[master]$ git grep -c subscrition
ovh/data_cloud_project_database_log_subscription.go:3
ovh/data_cloud_project_database_log_subscriptions.go:1
ovh/resource_cloud_project_database_log_subscription.go:6
ovh/resource_cloud_project_region_loadbalancer_log_subscription.go:6
website/docs/d/cloud_project_database_log_subscriptions.html.markdown:1
website/docs/r/cloud_project_database_log_subscription.html.markdown:1
```

I guess we have a few to fix